File: datasets_v2/finance/gnucash/compose/account_balance_toolbar.kt
import androidx.compose.ui.tooling.preview.Preview
import androidx.compose.foundation.background
import androidx.compose.foundation.layout.*
import androidx.compose.foundation.lazy.LazyColumn
import androidx.compose.foundation.lazy.items
import androidx.compose.material3.*
import androidx.compose.runtime.*
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.paint
import androidx.compose.ui.graphics.Color
import androidx.compose.foundation.Image
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.layout.ContentScale
import androidx.compose.ui.text.style.TextAlign
import androidx.compose.ui.unit.sp
import androidx.compose.ui.unit.dp

@Composable
fun account_balance_toolbar() {
  Row(modifier = Modifier.paint(painterResource(id = R.drawable.abc_ab_share_pack_mtrl_alpha), contentScale = ContentScale.Crop).fillMaxWidth().wrapContentHeight().padding(top = 8.0.dp).padding(bottom = 8.0.dp), verticalAlignment = Alignment.CenterVertically) {
    Text(text = "Balance:", modifier = Modifier.wrapContentHeight().weight(1.0f).padding(start = 12.0.dp), color = Color.Black, fontSize = 20.sp, textAlign = TextAlign.Center)
    Text(text = "", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(end = 12.0.dp), color = Color.Black, fontSize = 20.sp, textAlign = TextAlign.Center)
  }
}


@Preview(showBackground = true)
@Composable
fun account_balance_toolbar_preview() {
    MyApplicationTheme {
        account_balance_toolbar()
    }
}

File: datasets_v2/finance/gnucash/swiftui/Account_balance_toolbar.swift
import SwiftUI

struct Account_balance_toolbar: View {
  var body: some View {
    HStack(alignment: .center, spacing: 0) {
      Text("Balance:").font(.system(size: 20)).foregroundColor(Color.black).multilineTextAlignment(.center).frame(maxWidth: .infinity, alignment: .center).frame(maxWidth: .infinity).padding(.leading, 12.0)
      Text("").font(.system(size: 20)).foregroundColor(Color.black).multilineTextAlignment(.center).padding(.trailing, 12.0)
    }
    .background(Image("abc_ab_share_pack_mtrl_alpha").resizable().scaledToFill())
    .frame(maxWidth: .infinity)
    .padding(.top, 8.0)
    .padding(.bottom, 8.0)
  }
}

struct Account_balance_toolbar_Previews: PreviewProvider {
  static var previews: some View {
    Account_balance_toolbar()
  }
}
